backend/offchain/types/__init__.py

```diff
@@ -355,9 +363,161 @@ def new_payment_info_object(
     )
 
 
+def new_init_auth_command(
+    reference_id,
+    vasp_address,
+    sender_name,
+    sender_sure_name,
+    sender_city,
+    sender_country,
+    sender_line1,
+    sender_line2,
+    sender_postal_code,
+    sender_state,
+    sender_national_id_value,
+    sender_national_id_type,
+):
+    return CommandRequestObject(
+        cid=reference_id or str(uuid.uuid4()),
+        command_type=CommandType.InitAuthorizeCommand,
+        command=InitAuthorizeCommand(
+            reference_id=reference_id,
```

Comment:
From line 381 I understand that `reference_id` could be None, but then here it is assigned as if it isn't. Am I missing something?